### ISSUE TITLE
Cannot generate documentation with debt 0.20.0

### ISSUE THREAD

**Issue by cscetbon** [codeforkjeff/dbt-sqlite#11]
@codeforkjeff I found another issue and I think it comes from the upgrade. What do you think ?
```
Running with dbt=0.20.0
Found 5 models, 19 tests, 0 snapshots, 0 analyses, 538 macros, 0 operations, 2 seed files, 0 sources, 0 exposures

03:54:44 | Concurrency: 1 threads (target='dev')
03:54:44 |
03:54:45 | Done.
03:54:45 | Building catalog
Encountered an error while generating catalog: Database Error
  no such table: main_dbt_test__audit.sqlite_master
dbt encountered 1 failure while writing the catalog
03:54:45 | Catalog written to /dbt/target/catalog.json
```
Same as before your changes for the other commands it uses a wrong database name instead of just `main`

**Comment by codeforkjeff** [comment]
Thanks for reporting this. I released version 0.2.1 with a fix and just published it on pypi.

**Comment by cscetbon** [comment]
@codeforkjeff I can generate the documentation, however when I try to load the index file it can't process the manifest and catalog even if they are the same level.
<img width="416" alt="Screen Shot 2021-08-25 at 11 49 00 AM" src="https://user-images.githubusercontent.com/2619840/130822968-cd6501f6-0d7a-41eb-8f3a-0345f8e794e9.png">
<img width="931" alt="Screen Shot 2021-08-25 at 11 49 19 AM" src="https://user-images.githubusercontent.com/2619840/130823019-a6474819-50c7-4755-bed4-89b3a08f993d.png"
